### PR TITLE
[MINOR] Add log to show where we're reading the jwt from

### PR DESCRIPTION
**Pull request by gfukushima**
## PR description
Add a log to print the path where we loaded the engine jwt for better ux.

### CODE REVIEW

File: besu/src/main/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilder.java [hunk 1]
```diff
@@ -142,13 +143,24 @@ public ConfigurationOverviewBuilder setEngineApis(final Collection<String> engin
...
     return this;
   }
 
+  /**
+   * Sets the engine jwt file path.
+   *
+   * @param engineJwtFilePath the engine apis
+   * @return the builder
+   */
+  public ConfigurationOverviewBuilder setEngineJwtFile(final String engineJwtFilePath) {
```
Comment (siladu) [inline besu/src/main/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilder.java:159]: nit: could use Path or File here to make this even safer, e.g. if somehow the secret token ended up getting passed in by mistake

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java [hunk 2]
```diff
@@ -3630,10 +3630,17 @@ private String generateConfigurationOverview() {
           .setRpcHttpApis(jsonRpcConfiguration.getRpcApis());
     }
 
-    if (engineJsonRpcConfiguration != null && engineJsonRpcConfiguration.isEnabled()) {
-      builder
-          .setEnginePort(engineJsonRpcConfiguration.getPort())
-          .setEngineApis(engineJsonRpcConfiguration.getRpcApis());
+    if (engineJsonRpcConfiguration != null) {
+      if (engineJsonRpcConfiguration.isEnabled()) {
```
Comment (siladu) [inline besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:3634]: Why are these broken up?

I'm struggling to imagine a scenario where jwt token is needed without the engine api being enabled.

**Review by ajsutton — APPROVED** [review]
LGTM.

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java [hunk 3]
```diff
@@ -3630,10 +3630,17 @@ private String generateConfigurationOverview() {
...
+        builder
+            .setEnginePort(engineJsonRpcConfiguration.getPort())
+            .setEngineApis(engineJsonRpcConfiguration.getRpcApis());
+      }
+      if (engineJsonRpcConfiguration.isAuthenticationEnabled()
+          && engineJsonRpcConfiguration.getAuthenticationPublicKeyFile() != null) {
```
Comment (siladu) [inline besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:3640]: If someone mistakenly doesn't configure it, what will we print? I think there may be some other error so probably ok

Comment (gfukushima) [reply besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:3640]: Prints nothing, engineJsonRpcConfiguration.getAuthenticationPublicKeyFile() will be null

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java (hunk 2 above)
Comment (gfukushima) [reply besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:3634]: Think you're right. Refactored this. Thank you

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java (hunk 3 above)
Comment (siladu) [reply besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:3638]: ah, it generates one and logs a warning:
`2023-03-29 13:31:29.817+10:00 | main | WARN  | Besu | Engine API authentication enabled without key file. Expect ephemeral jwt.hex file in datadir`

Do you think we should also print out the location of this ephemeral one in the configuration overview?

**Review by siladu — APPROVED** [review]
LGTM

**Pull request merged**